src/redux/rockets/rocketsSlice.js

```diff
@@ -0,0 +1,14 @@
+import { createSlice } from '@reduxjs/toolkit';
+
+const initialState = {
+  Rockets: [],
+};
+
+const rocketsSlice = createSlice({
+  name: 'rockets',
+  initialState,
+  reducers: {},
+});
+
+export const { actions, recuder } = rocketsSlice;
+export default rocketsSlice.reducer;
```

Comment:
Please Add action to add a rocket to make sure it works,

Also add the rocketslice to the store.